index.html

```diff
@@ -1412,6 +1412,104 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID method, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      The <a 
+      href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG
+      Ethical Web Principles</a> document is a reliable guide for making assessments
```

Comment:
```suggestion
      Ethical Web Principles</a> document is a reliable guide for making assessments
```

I'd get rid of this first part. The Ethical Web Principles is not a guide for making assessments, much less a "reliable guide". I'd just stick with the balance of the sentence "The ... Principles are a guide for weighing ethical considerations..."

Reply:
adjustment to language from @TallTed committed